app/models/html_attachment.rb

```diff
@@ -92,6 +92,10 @@ def save_and_update_publishing_api
     save && Whitehall.edition_services.draft_updater(attachable).perform!
   end
 
+  def translated_locales
```

Comment:
Tells the `PublishingAPIWorker` which locales to send to the publishing api. Set by default for Editions, but HTML Attachments aren't Editions.